tutor/templates/build/openedx/Dockerfile

```diff
@@ -92,6 +92,10 @@ ENTRYPOINT ["docker-entrypoint.sh"]
 # Copy gunicorn settings file
 COPY gunicorn_conf.py /openedx/gunicorn_conf.py
 
+RUN mkdir /openedx/logs \
+    && ln -s /dev/stdout /openedx/logs/tracking.log \
+    && ln -s /dev/stderr /openedx/logs/edx.log
```

Comment:
The problem with this is that, by default, non-tracking logs will be printed twice in the console. I propose to replace `/dev/stdout` and `/dev/stderr` by `/dev/null`.